src/app/(main)/my-voyage/[teamId]/sprints/components/sections/Notes.tsx

```diff
@@ -3,31 +3,82 @@
 import { z } from "zod";
 import { SubmitHandler, useForm } from "react-hook-form";
 import { zodResolver } from "@hookform/resolvers/zod";
+import { useParams } from "next/navigation";
 
 import Textarea from "@/components/inputs/Textarea";
 import Button from "@/components/Button";
 
 import { validateTextInput } from "@/helpers/form/validateInput";
+import useServerAction from "@/hooks/useServerAction";
+import { editMeeting } from "@/myVoyage/sprints/sprintsService";
+import { useAppDispatch } from "@/store/hooks";
+import { onOpenModal } from "@/store/features/modal/modalSlice";
 
 const validationSchema = z.object({
   notes: validateTextInput({
-    inputName: "notes",
+    inputName: "This field",
     required: true,
   }),
 });
 
 export type ValidationSchema = z.infer<typeof validationSchema>;
 
-export default function Notes() {
+interface NotesProps {
+  data?: string;
+}
+
+export default function Notes({ data }: NotesProps) {
+  const dispatch = useAppDispatch();
+  const params = useParams<{
+    sprintNumber: string;
+    meetingId: string;
+  }>();
+
+  const [sprintNumber, meetingId] = [
+    Number(params.sprintNumber),
+    Number(params.meetingId),
+  ];
+
   const {
     register,
     handleSubmit,
+    reset,
     formState: { errors, isDirty, isValid },
   } = useForm<ValidationSchema>({
+    mode: "onTouched",
     resolver: zodResolver(validationSchema),
+    defaultValues: {
+      notes: data ?? "",
+    },
```

Comment:
So I haven't been able to see what the need for this is. I tried removing it and adding the default value to the input itself and it seems to work fine. If you can look into this again and see why you needed this, that'd be great. Same applies to all the other sections too.

Reply:
I've decided to use `useEffect` to reset values like in all other forms.